src/locales/en/translation.json

```diff
@@ -95,9 +95,10 @@
     "undefined": "Undefined",
     "unknown": "Unknown",
     "value": "Value",
-    "valueInToken": "{{value}} {{ticker}}",
-    "roundedValueInToken": "{{value}}… <TickerLink />",
-    "valueInTokenWithLink": "{{value}} <TickerLink />",
+    "valueLong": "{{value, number}}",
```

Comment:
Here's what happens if we add a language switcher:
| English | French |
| --- | --- |
| ![localhost_1234_mainnet_sapphire_address_0xE8D562606F35CB14dA3E8faB1174F9B5AE8319c4_tokens_erc-20 (1)](https://github.com/oasisprotocol/explorer/assets/3758846/3c282d2a-74e2-4d0b-8314-aaee52bac0fd)  | ![localhost_1234_mainnet_sapphire_address_0xE8D562606F35CB14dA3E8faB1174F9B5AE8319c4_tokens_erc-20](https://github.com/oasisprotocol/explorer/assets/3758846/9a0c62c8-da82-4878-9def-1ce31cfedbf4)  |